libsql/internal/http/hranaV2/hranaV2.go

```diff
@@ -477,7 +579,7 @@ func (h *hranaV2Conn) QueryContext(ctx context.Context, query string, args []dri
 	}
 }
 
-func (h *hranaV2Conn) ResetSession(ctx context.Context) error {
```

Comment:
Hm..why not `resetSession`?

Reply:
It's part of the database/sql connection interface